altair/examples/dendrogram.py

```diff
@@ -0,0 +1,131 @@
+"""
+Dendrogram of Hierarchical Clustering
+-------------------------------------
+This is a dendrogram from the result of a hierarchical clustering. It's based on the example from
+https://scikit-learn.org/stable/auto_examples/cluster/plot_agglomerative_dendrogram.html
+"""
+# category: case studies
+
+import pandas as pd
+import altair as alt
+
+den = {
```

Comment:
I wonder if it would be worthwhile adding a comment about how these coordinates can be obtained from sklearn (or a similar library), even if we don't use that code in the docs directly.

Reply:
Added some comments to explain where the input data comes from.